internal/kldevents/eventstream.go

```diff
@@ -65,6 +68,7 @@ type StreamInfo struct {
 	RetryTimeoutSec      uint64         `json:"retryTimeoutSec,omitempty"`
 	BlockedRetryDelaySec uint64         `json:"blockedReryDelaySec,omitempty"`
 	Webhook              *webhookAction `json:"webhook,omitempty"`
+	Timestamps           bool           `json:"timestamps,omitempty"` // Include block timestamps in the events generated
```

Comment:
I was looking for the change to the `update` function. Not sure if I missed it in the change history?

Reply:
Duh, missed the `updateStream` change. Will add that